Cluster.php

```diff
@@ -10,335 +10,198 @@
  */
 
 /**
- * A generalized Credis_Client interface for a cluster of Redis servers
+ * Credis_Cluster, subclass to Credis_Client that uses RedisCluster (in phpredis extension)
  *
- * @deprecated
+ * Note: RedisCluster currently has limitations like not supporting pipeline or multi.
+ * Note: Many methods require an additional parameter to specify which node to run on, and only run on that node,
```

Comment:
I need to update the comments here after the changes that I've made in last couple of days.

Reply:
resolved in 37de734eba6e246dc4eb2c40189fb95e3443b8ae